### DISCUSSION TITLE
Requisição com API_KEY

### DISCUSSION THREAD

**Discussion by ViniciusSilvaToqSys** [viniciussanchez/RESTRequest4Delphi#128] [General]
Olá, é possivel fazer requisições onde a API tem uma access_token via api_key? 
Eu fiz assim

```
...
.AddHeader('access_token', API_KEY); // API_KEY é uma constante com minha key 
...
```

ta dando erro 401, tipo como se não tivesse enviando minha api_key correta..

Agardeço qualquer ajuda.

**Comment by leonardo-abc** [comment]
Bom dia meu amigo, você resolveu essa questão? Estou com esse mesmo problema.
